etabotsite/etabotapp/permissions.py

```diff
@@ -4,6 +4,16 @@
 from django.contrib.auth.models import User
 
 
+class IsOwner(permissions.BasePermission):
+    """Custom permission class to allow only bucketlist owners to edit them."""
+    def has_permission(self, request, view):
+        return request.user
```

Comment:
is this meant to return True or False?

Reply:
It returns `True` if `request.user` exist. Otherwise, `False`